recipes/gidgethub/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "gidgethub" %}
+{% set version = "2.5.0.post1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7949b92337aeb95d28ef019e34600778de3e84eebce06efb7e4577a383c42375
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  host:
```

Comment:
I'm curious as to why the build isn't failing since `pytest-runner` is listed under `setup_requires` in the `setup.py`

Also, I believe you need `'uritemplate>=3.0.0`. It doesn't get imported in your tests, but it looks to be needed.

Reply:
It looks like the `__init__.py` doesn't actually import from the submodules. 

I don't know about pytest-runner. It seems like it should only be a test requires.